README.md

```diff
@@ -16,18 +16,15 @@ See the full list of
 [officially-supported](https://appium.io/docs/en/latest/ecosystem/#drivers) and
 [third-party](https://appium.io/docs/en/latest/ecosystem/#other-drivers) drivers for more details.
 
-:bangbang: Major documentation revision in progress
-
-:bangbang: Appium core team does not maintain Appium 1.x anymore since the 1st of January 2022. All recent versions of officially supported platform drivers are not compatible to Appium 1.x anymore, and require Appium 2 to run. [Please read the migration guide from 1.x to 2.0](https://appium.io/docs/en/latest/guides/migrating-1-to-2/) to manage the Appium server.
-
-Appium is in the final stages of a major revision (to version 2.0). As such, the documentation
-found around the web may not be correct. The current Appium 2.0 documentation is very much in
+:bangbang: The current Appium 2.0 documentation is very much in
```

Comment:
I don't think it's much in progress anymore. I think we can just link to it.

Reply:
changed